docs/quick-starts/feature-flag-quickstart.md

```diff
@@ -7,7 +7,7 @@ sidebar_position: 3
 
```

Comment:
One downside of this page is that, while the example is practically relevant, I cannot actually run the code locally to see that everything is set up correctly.

It would be a bit of work, but I think there could be a lot of value in actually creating a walkthrough that creates something that runs locally (probably a little Python script is the easiest). We could start using "print" to log assignments, and then explain how to hook up Segment as a follow up step.